cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -130,18 +130,24 @@ func (command *Evaluate) Execute(args []string) (err error) {
 	{
 		models := map[string]model.Model{}
 		for _, p := range provider.Providers {
+			if t, ok := p.(provider.InjectToken); ok {
```

Comment:
For whatever reason https://github.com/symflower/eval-dev-quality/actions/runs/9042053222/job/24848117624?pr=95 fails in "Install test tools" but does not fail. Fix this first before moving on because CI should fail on failures even if Windows does not think so.

Reply:
Fixed with early merger.